src/main/java/org/folio/circulation/domain/notice/NoticeDescriptorBuilder.java

```diff
@@ -0,0 +1,55 @@
+package org.folio.circulation.domain.notice;
+
+import org.joda.time.Period;
+
+public class NoticeDescriptorBuilder {
```

Comment:
This needs renaming now the class it creates has been renamed

Reply:
Done